content/docs/get-started/gcp/create-project.md

```diff
@@ -2,7 +2,7 @@
 title: Create a New Project | GCP
 h1: Create a New Project
 linktitle: Create a New Project
-meta_desc: This page provides an overview of how to create a new Google Cloud (GCP) + Pulumi project.
```

Comment:
cc @zchase, as these may be here for SEO usefulness. (Although I suspect Google's probably smart enough to relate searches for "GCP" to include "Google Cloud". 😄

Reply:
I've removed most of the inclusions to "GCP" from our docs and metadata because Google themselves no longer use the acronym GCP (and any downstream acronyms like GCS to refer to Google Cloud Storage). Of course, GCP is still used many places on cloud.google.com, so the rollout of this change will likely take a few months.